book/src/tutorial_with_flutter.md

```diff
@@ -24,6 +24,21 @@ Please [install Flutter](https://flutter.dev/docs/get-started/install) (optional
 git clone https://github.com/fzyzcjy/flutter_rust_bridge && cd flutter_rust_bridge/frb_example/with_flutter
 ```
 
+## Setup dependencies
+
+For non-debian based distributions a few tricks required.
+Ensure that packages are up to date (or install by demand).
+
+- clang
+- llvm-libs
+- glibc
+
+Set environment variable in your shell profile (`.bashrc`, `.zshrc`, etc).
+
+```bash
+export CPATH="$(clang -v 2>&1 | grep "Selected GCC installation" | rev | cut -d' ' -f1 | rev)/include"
+```
```

Comment:
So, you mentioned "non-debian" users. This includes:

- [ ] windows
- [ ] linux (except debian)
- [ ] macos (M1, Intel)

Is it true that all these kinds of users need the setup? (IMHO I myself do not need it; I am on macos intel)

Reply:
You are right, I mean Linux other than debian. At least during issues research I have not met others